ui/tests/integration/components/agent-monitor-test.js

```diff
@@ -72,13 +72,15 @@ module('Integration | Component | agent-monitor', function(hooks) {
     assert.ok(logRequest.url.startsWith('/v1/agent/monitor'));
     assert.ok(logRequest.url.includes('client_id=client1'));
     assert.ok(logRequest.url.includes('log_level=info'));
+    assert.ok(logRequest.url.includes('log_level=info'));
```

Comment:
Is this meant to be a different assertion? Or accidental copypaste? 😯

Reply:
I think this was vim betraying me? Dunno, but I'll clean it up.